packages/graphql-hooks/test/unit/GraphQLClient.test.js

```diff
@@ -3,7 +3,9 @@ import { Readable } from 'stream'
 import { FormData, File as FormDataFile } from 'formdata-node'
 import { GraphQLClient } from '../../src'
 import { createReadStream } from 'fs'
-
+import { TextEncoder, TextDecoder } from 'util'
```

Comment:
nit: keep import statement together but put a new line between imports and other statements